src/components/RenderConfig/RenderConfigComponent.tsx

```diff
@@ -254,7 +254,7 @@ export class RenderConfigComponent extends React.Component<WidgetProps> {
 
         let unitString = "Value";
         if (frame && frame.unit) {
```

Comment:
shall we check `requiredUnit` instead of `unit`?

Reply:
fixed.